dashboards/opensearch/opensearch_details.sp

```diff
@@ -0,0 +1,223 @@
+# locals.tf
+
+locals {
+  opensearch_common_tags = {
+    service = "AWS/OpenSearch"
+  }
+}
```

Comment:
Can you please check the detail pages of the other services and incorporate them into the dashboards/opensearch/opensearch.sp file to ensure consistency? Thanks.

Reply:
@khushboo9024  thanks 
Its updated